samples/core/blur/main.c

```diff
@@ -1114,22 +1129,25 @@ int main(int argc, char *argv[])
                          error, prg);
 
         /// Subgroup exchange in dual-pass blur
-        if (use_subgroup_exchange_relative)
+        if (use_subgroup_exchange_relative && opencl_version_2_0)
         {
             printf("Dual-pass subgroup relative exchange blur\n");
 
             kernel_op[0] = '\0';
+            // cl_khr_subgroup_shuffle_relative requires OpenCL 2.0
+            strcat(kernel_op, " -cl-std=CL2.0 ");
             strcat(kernel_op, "-D USE_SUBGROUP_EXCHANGE_RELATIVE ");
-
             OCLERROR_RET(dual_pass_subgroup_exchange_box_blur(
                              &s, (cl_int)blur_opts.size),
                          error, prg);
         }
-        if (use_subgroup_exchange)
+        if (use_subgroup_exchange && opencl_version_2_0)
         {
             printf("Dual-pass subgroup exchange blur\n");
 
             kernel_op[0] = '\0';
+            // cl_khr_subgroup_shuffle requires OpenCL 2.0
+            strcat(kernel_op, " -cl-std=CL2.0 ");
```

Comment:
This is also incorrect and needs to be fixed: there is no dependency between `cl_khr_subgroup_shuffle` and OpenCL C 2.0.